jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/internal/HttpChannelState.java

```diff
@@ -1199,14 +1193,13 @@ public void succeeded()
         {
             if (LOG.isDebugEnabled())
                 LOG.debug("write succeeded {}", this);
-            // Called when an individual write succeeds.
```

Comment:
Why eliminate these comments?

Reply:
Because they add nothing and repeat exactly what's written in the log the line above.